### PR TITLE
Blockchain/Wallet Service/Client

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Blockchain/Wallet Service/Client

### CODE REVIEW

**Comment by mod** [comment]
Can you explain why you see it here ?

**Comment by dinesh-skyach** [comment]
@mod, It's better to move the Blockchain/Wallet Service files to peatio-core. Then we will inherit to peatio and coin plugins.

This is how it was discussed with @ysv.

File: lib/peatio/client/blockchain_client.rb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      def create_withdrawal!(issuer, recipient, amount, options = {})
+        method_not_implemented
+      end
+
+      # TODO: Doc.
+      def inspect_address!(address)
+        method_not_implemented
+      end
+
+      def convert_to_base_unit!(value)
```
Comment (mod) [inline lib/peatio/client/blockchain_client.rb]: I think we need to make a fully Abstract Class here
The plugin api principle should be to implement Abstract class and include only peatio-core
@dinesh-skyach @ysv @ritesh-skyach

Comment (dinesh-skyach) [reply lib/peatio/client/blockchain_client.rb]: @mod I believe we can convert the `Base` class to module and then we include the module in plugins. What do you say?

**Pull request closed without merging**